packages-backend/express/tsconfig.json

```diff
@@ -1,3 +1,3 @@
 {
-  "extends": "@tsconfig/node16/tsconfig.json"
+  "extends": "@tsconfig/node18/tsconfig.json"
```

Comment:
I was also wondering if we should keep TS configuration linked to `v16`, mostly because I'm afraid using `v18` could potentially apply some code transformation not supported in `v16`.

Reply:
Fair enough, let's keep it at `v16` for now then. Thanks for pointing it out 😊

Reply:
https://github.com/commercetools/merchant-center-application-kit/pull/3039/commits/293b5d7c2f3b177eddfa1d33002f7f8cab46ae91